src/MezzTest.cpp

```diff
@@ -213,11 +213,13 @@ namespace Mezzanine
             {
                 OneTestGroup(); // Run tests and discard results, the parent process will grab it.
             } else {
-                String ProcessLog = RunCommand(
+                String LogFileName = OneTestGroup.Name() + "_SubProcess.log";
+                OutputCommandToFile(
```

Comment:
I just traced the logic for code that decides whether or not we are in child and I do not see a way to call this in a test without an unreasonable amount of complexity.